core/src/de/adesso/brainysnake/BrainySnake.java

```diff
@@ -56,7 +83,8 @@ public void create() {
 
         game = new Game();
         game.init(HEIGHT, WIDTH);
-        Gdx.input.setInputProcessor(new KeyBoardControl(game));
+
```

Comment:
Good bye KeyboardControl for KeyBoardControl-Snake ;) Pls change the Input Processor within gameplay to keyboard.

Reply:
Usage of InputMultiplexer to add multiple InputProcessors.

Reply:
Top